pixl_core/pyproject.toml

```diff
@@ -7,36 +7,40 @@ readme = "README.md"
 requires-python = ">=3.9"
 classifiers = ["Programming Language :: Python :: 3"]
 dependencies = [
+    "aio_pika==9.4.2",
     "azure-identity==1.16.1",
     "azure-keyvault==4.2.0",
     "fastapi==0.112.0",
-    "token-bucket==0.3.0",
-    "python-decouple==3.8",
-    "python-slugify==8.0.4",
-    "pika==1.3.2",
-    "aio_pika==9.4.2",
-    "requests==2.32.2",
     "jsonpickle==3.2.2",
     "loguru==0.7.2",
-    "sqlalchemy==2.0.32",
-    "psycopg2-binary==2.9.9",
```

Comment:
I put the dependencies in alphabetical order to make it easier to see which ones are duplicated

Reply:
Yeah good call